utils/config.go

```diff
@@ -434,6 +434,7 @@ func getClientConfig(c *model.Config) map[string]string {
 	props["EnableSignInWithUsername"] = strconv.FormatBool(*c.EmailSettings.EnableSignInWithUsername)
 	props["RequireEmailVerification"] = strconv.FormatBool(c.EmailSettings.RequireEmailVerification)
 	props["EnableEmailBatching"] = strconv.FormatBool(*c.EmailSettings.EnableEmailBatching)
+	props["EnableSMTPAuth"] = strconv.FormatBool(*c.EmailSettings.EnableSMTPAuth)
```

Comment:
This shouldn't be in the client config since it only contains settings needed by the client outside of the System Console